core/app/models/spree/promotion.rb

```diff
@@ -40,6 +40,7 @@ class Promotion < Spree::Base
         where(table[:expires_at].eq(nil).or(table[:expires_at].gt(time)))
     end
     scope :applied, -> { joins(:order_promotions).uniq }
+    scope :applied_automatically, -> { where(apply_automatically: true) }
```

Comment:
How about `apply_automatically` instead of `applied_automatically` here?  `applied_xxx` feels a bit confusing given that we have an `applied` scope that means something very different.